apps/mobile/src/features/send-refactor/screens/form.tsx

```diff
@@ -0,0 +1,118 @@
+import { useRef, useState } from 'react';
+
+import { FullHeightSheetHeader } from '@/components/full-height-sheet/full-height-sheet-header';
+import { FullHeightSheetLayout } from '@/components/full-height-sheet/full-height-sheet.layout';
+import { HeaderBackButton } from '@/components/headers/components/header-back-button';
+import { InlineAssetPicker } from '@/features/send-refactor/components/inline-asset-picker';
+import { BtcForm } from '@/features/send-refactor/forms/btc/btc-form';
+import { BtcDataLoader } from '@/features/send-refactor/forms/btc/btc-loader';
+import { StxForm } from '@/features/send-refactor/forms/stx/stx-form';
+import { StxDataLoader } from '@/features/send-refactor/forms/stx/stx-loader';
+import { useSendNavigation, useSendRoute } from '@/features/send-refactor/navigation';
+import { useSendFlowContext } from '@/features/send-refactor/send-flow-provider';
+import { SendableAsset } from '@/features/send-refactor/types';
+import { NetworkBadge } from '@/features/settings/network-badge';
+import { useSettings } from '@/store/settings/settings';
+import { i18n } from '@lingui/core';
+import { t } from '@lingui/macro';
+
+import { SheetRef } from '@leather.io/ui/native';
+
+export function Form() {
+  const { canGoBack, goBack } = useSendNavigation();
+  const { params } = useSendRoute<'form'>();
+  const {
+    state: { selectedAsset, selectedAccount },
+    selectAsset,
+  } = useSendFlowContext();
+  const { fiatCurrencyPreference } = useSettings();
+  const assetPickerSheetRef = useRef<SheetRef>(null);
+  const [shouldAnimateAssetItem, setShouldAnimateAssetItem] = useState(true);
+  const assetItemElementInitialOffset = shouldAnimateAssetItem
+    ? params?.assetItemElementInitialOffset
+    : undefined;
+
+  if (!selectedAsset || !selectedAccount) {
+    return null;
+  }
+
+  function handleOpenAssetPicker() {
+    assetPickerSheetRef.current?.present();
+  }
+
+  function handleInlineAssetSelection(asset: SendableAsset) {
+    setShouldAnimateAssetItem(false);
+    selectAsset(asset);
+  }
+
+  return (
+    <>
+      <FullHeightSheetLayout
+        header={
+          <FullHeightSheetHeader
+            title={t({
+              id: 'send_form.header_title',
+              message: 'Send',
+            })}
+            subtitle={i18n._({
+              id: 'select_asset.header_subtitle',
+              message: '{subtitle}',
+              values: { subtitle: selectedAccount.name },
+            })}
+            leftElement={canGoBack() ? <HeaderBackButton onPress={goBack} /> : null}
+            rightElement={<NetworkBadge />}
+          />
+        }
+      >
+        {/* TODO: Use pattern matching once we add support for remaining tokens */}
```

Comment:
I wouldn't bother with these comments as I'm not sure when we will do this and can figure it out when we do